transport/http/codes.go

```diff
@@ -20,16 +20,11 @@
 
 package http
 
-import (
-	"fmt"
-
-	"go.uber.org/yarpc/api/yarpcerrors"
-)
-
-// TODO: Should we expose the maps as public variables to document the mappings?
+import "go.uber.org/yarpc/api/yarpcerrors"
 
 var (
-	_codeToHTTPStatusCode = map[yarpcerrors.Code]int{
+	// CodeToStatusCode maps all Codes to their corresponding HTTP status code.
+	CodeToStatusCode = map[yarpcerrors.Code]int{
```

Comment:
Ya it's annoying, however I think some linters will fail if you modify a public global variable? I could be wrong, just going off of memory.